components/carousel/index.tsx

```diff
@@ -70,6 +71,12 @@ export default class Carousel extends React.Component<CarouselProps, {}> {
     this.innerSlider = this.slick && this.slick.innerSlider;
   }
 
+  componentDidUpdate(prevProps: CarouselProps) {
+    if (React.Children.count(this.props.children) !== React.Children.count(prevProps.children)) {
+      this.goTo(0, false);
```

Comment:
react-slick can set `initialSlide` prop，should `goTo(0)`  be set to ` goTo(initialSlide)`?  https://react-slick.neostack.com/docs/api#initialSlide

Reply:
Greate~ Could you help to send a PR about this?